uniffi_bindgen/src/bindings/kotlin/gen_kotlin/mod.rs

```diff
@@ -576,6 +576,15 @@ impl<T: AsType> AsCodeType for T {
     }
 }
 
+// A work around for #2392 - we can't handle functions with external errors.
+fn can_render_callable(callable: &dyn Callable, ci: &ComponentInterface) -> bool {
```

Comment:
@bendk this is what I was thinking re our discussions around tests etc - at some point it seems reasonable for binding generators to simply decline to render things they can't handle - eg, I could imagine a new bindgen might start by declining to implement functions which try and use anything other than primitive types as they are starting out etc.